src/main/java/com/conveyal/datatools/manager/models/Project.java

```diff
@@ -61,6 +61,10 @@
     // and/or for applying a geographic filter when syncing with external feed registries.
     public Bounds bounds;
 
+    /**
+     * Whether the {@link #pinnedDeploymentId} should be re-deployed after a new feed version is successfully processed.
+     */
+    public boolean autoDeploy;
```

Comment:
There is nothing built out in the UI repo that could set this to true. It'd be nice to have at least a PR in https://github.com/ibi-group/datatools-ui/ that allows us to try out this new setting.